pyface/ui/qt4/util/gui_test_assistant.py

```diff
@@ -0,0 +1,242 @@
+# Copyright (c) 2013-2015 by Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in enthought/LICENSE.txt and may be redistributed only
+# under the conditions described in the aforementioned license.  The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+# Thanks for using Enthought open source!
+
+import contextlib
+import threading
+
+import mock
+
+from pyface.qt.QtGui import QApplication
+from pyface.ui.qt4.gui import GUI
+from traits.testing.unittest_tools import UnittestTools
+from traits.testing.unittest_tools import _TraitsChangeCollector as \
+    TraitsChangeCollector
+
+from .testing import find_qt_widget, print_qt_widget_tree
```

Comment:
It's there because the internal codebase that this code was pulled from had this function available in this module, and it will make it easier to translate code if we keep it here, even though it is not used.